### PR TITLE
(segmentation codebase)during training, actual lr is not set by args.lr

### PR DESCRIPTION
**Pull request by fortunechen**
initial `learning rate ` will be set as 0.01  in [optimizer.py](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/optimizer/optimizer.py)  if we don't pass `'learning_rate': args.lr` in `optimizer_params`

### CODE REVIEW

**Comment by mli** [comment]
Job PR-776-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-776/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-776-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-776/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-776-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-776/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhanghang1989** [comment]
loss is a scalar, we just need to get the value. Why use np.mean ?

**Comment by fortunechen** [comment]
> loss is a scalar, we just need to get the value. Why use np.mean ?

I think loss is a vector, the dimension is the batch_size on each gpu as defined in `return F.mean(loss, axis=self._batch_axis, exclude=True)` [loss.py line 284](https://github.com/dmlc/gluon-cv/blob/master/gluoncv/loss.py). 

Suppose we have 4gpus and the training batch size is 16.
`train_loss += loss.asnumpy()[0] / len(losses)` only considers the first example loss on every gpu, so the `train_loss` is average loss of 4 examples  rather than 16 examples.

**Comment by zhanghang1989** [comment]
I see. You're right.
